docs/advanced_usage.rst

```diff
@@ -128,12 +128,12 @@ priority to the following values:
 - ``AUTO_REJECT``: the sampler automatically rejects the trace
 - ``AUTO_KEEP``: the sampler automatically keeps the trace
 
-Priority sampling is disabled by default. Enabling it ensures that your sampled
-distributed traces will be complete. To enable priority sampling::
+Priority sampling is enabled by default. Enabling it ensures that your sampled
+distributed traces will be complete. To disable priority sampling::
 
-    tracer.configure(priority_sampling=True)
+    tracer.configure(priority_sampling=False)
```

Comment:
Let's remove this snippet. Disabling priority sampling, means breaking distributed tracing so I'd say just to tell people how to use it.